src/Bex/Behat/ScreenshotExtension/Listener/ScreenshotListener.php

```diff
@@ -81,11 +81,18 @@ public function takeScreenshot(AfterStepTested $event)
     {
         if ($this->shouldTakeScreenshot($event)) {
             $this->screenshotTaker->takeScreenshot();
+
+            if (!$this->config->shouldCombineImages()) {
```

Comment:
This change would upload a combined image after each step when screenshot taking mode is `failed_scenarios`.